release-notes/opensearch-release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,606 @@
+# OpenSearch and Dashboards 2.0.0-rc1 Release Notes
+
+## Release Highlights
+
+### OpenSearch Notifications
+* OpenSearch 2.0.0 is the first official release with OpenSearch Notifications.
+* Notifications consist of three plugins, `notifications-core` and `notifications` backend plugins for OpenSearch, and a `notificationsDashboards` frontend plugin for OpenSearch Dashboards.
+
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.0-rc1 include the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.0.0-rc1.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.0.0-rc1.md).
```

Comment:
The release notes for OpenSearch & OpenSearch Dashboards are not published yet. I put the potential links of them here.

Reply:
OS Core: https://github.com/opensearch-project/OpenSearch/blob/2.0/release-notes/opensearch.release-notes-2.0.0-rc1.md